libcontainer/configs/cgroup_linux.go

```diff
@@ -13,12 +13,14 @@ const (
 	Thawed    FreezerState = "THAWED"
 )
 
+// Cgroup holds properties of a cgroup on Linux
 type Cgroup struct {
-	// Deprecated, use Path instead
+	// Name specifies the name of the cgroup
+	// Deprecated: use Path instead
```

Comment:
I recently found out that Name and Parent are not deprecated at least for systemd cgroup manager case, because Path can not be used in that case. This might have to be fixed, but for now, I guess, it's better to remove the deprecation notice.